docs/source/datasets.rst

```diff
@@ -215,6 +215,19 @@ Vocal Activity
 A time series or sequence of events indicating when singing voice is present in a recording. This type
 of annotation is implicitly available when Vocal F0_ or Vocal Notes_ annotations are available.
 
+.. _stroke-name:
+
+Stroke Name
+^^^^^^^^^^^
+An open "tag" that refers to the original name of the stroke type that is played on a Mridangam instrument.
+
+.. _tonic:
+
+Tonic
+^^^^^^^^^^^
+Stored as metadata, the absolute tonic of a track. It is a useful metre for tracks with a single stroke or
+note. Originally added to describe the tonic of the Mridangam strokes.
```

Comment:
Same comment here about being general

Reply:
@magdalenafuentes should we define an ontology and use it everywhere?

Reply:
Yes! I also added a comment on #245 to explain this better in the docs later